sessionctx/variable/sysvar.go

```diff
@@ -1194,6 +1214,14 @@ var defaultSysVars = []*SysVar{
 	{Scope: ScopeGlobal | ScopeSession, Name: TiDBAnalyzeVersion, Value: strconv.Itoa(DefTiDBAnalyzeVersion), Type: TypeInt, MinValue: 1, MaxValue: 2},
 	{Scope: ScopeGlobal | ScopeSession, Name: TiDBEnableIndexMergeJoin, Value: BoolToOnOff(DefTiDBEnableIndexMergeJoin), Type: TypeBool},
 	{Scope: ScopeGlobal | ScopeSession, Name: TiDBTrackAggregateMemoryUsage, Value: BoolToOnOff(DefTiDBTrackAggregateMemoryUsage), Type: TypeBool},
+
+	/* tikv gc metrics */
+	{Scope: ScopeGlobal, Name: TiKVGCEnable, Value: BoolOn, Type: TypeBool},
+	{Scope: ScopeGlobal, Name: TiKVGCRunInterval, Value: "10m0s", Type: TypeDuration, MinValue: 0, MaxValue: math.MaxInt64},
```

Comment:
How about setting the `MinValue` to 10mins? Seems we only restrict it in the commend: 
> GC run interval, at least 10m, in Go format.